devel/fortran-utils/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            fortran-utils
+version         1.2
+revision        0
+categories      devel
+platforms       darwin
+license         BSD
+maintainers     {@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage        https://sourceforge.net/projects/fortran-utils/
+master_sites    sourceforge
+extract.suffix  .tar.bz2
+worksrcdir      {}
+checksums       rmd160  353e753b2084f3eab5cf9cfd9c2ccf4e983f05b5 \
+                sha256  b2a25ae867a7755fb8d839ca5dd59a24888d12d217aea6ddfd57aceda888b197 \
+                size    7689
+
+description     Tools for use with Fortran77 code
+
+long_description \
+    The fortran-utils are: \
+    \n- fpr: transform Fortran's carriage control conventions to UNIX line printer conventions \
+    \n- fsplit: split a multi-routine Fortran file into individual files
+
+notes "This version has patches from FreeBSD and NetBSD."
```

Comment:
I don't think there's any particular reason why we would need to mention that in notes, is there?
```suggestion
```

Reply:
Without the note, the origin of the patches remains unclear. It also makes clear that the patches from FreeBSD and NetBSD are included. For me, these are valid reasons for the note. Maybe, there is a better way to achieve this.

Reply:
Typically when you talk about patches, I think of the MacPorts `patchfiles` directive. This port doesn't use that, so we're not modifying the upstream code; we're just distributing it. Even if we were using `patchfiles` we wouldn't usually inform the user of it in notes or anywhere.

Are there different upstream versions of fortran-utils, some of which include these FreeBSD and NetBSD patches you mention and some of which don't?

Reply:
If there are other forks of fortran-utils 1.2 that don't include those patches, and if it's important to mention that this fork of fortran-utils 1.2 does, then I would mention it in the long_description not the notes.

Reply:
I checked the subversion repository and found my memories are wrong. It is actually obvious from the commit history of version 1.2 and can therefore be completely cleared.